tests/scripts/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:2.7
+
+RUN mkdir -p /src
+
+ADD requirements.txt tests/requirements-*.txt /src/
+RUN pip install \
+    -r /src/requirements.txt \
+    -r /src/requirements-tools.txt \
+    -r /src/requirements-libraries.txt
+
+ADD *.py setup.cfg README.rst MANIFEST.in .pylintrc /src/
+ADD PyInstaller /src/PyInstaller
+ADD bootloader /src/bootloader
+ADD doc /src/doc
```

Comment:
Is this used at all in the container? I assume this can be removed.

Reply:
I only added things which were required for one of these steps:

- `python setup.py install`
- `python waf distclean all` (in `bootloader/`)
- `py.test tests/functional`

I forget which step needed `doc/` to exist, but one of them did.